pkg/pipeline/pipeline_netobs_test.go

```diff
@@ -0,0 +1,201 @@
+package pipeline
```

Comment:
@mariomac if this is "specific" for some "use-case" or "scenario" ... maybe the right thing is to open a folder called scenarios and put the test file inside this folder. We might have more "use-cases" tests in the future. Also, can you think of some other name for "netobs" it is not informative? Maybe something like a redhat-ocp scenario or something like that. We need to think on this repo as being an upstream repo.

Reply:
This was located here because tests we often rely on private APIs. But you are right when you say that this is only a scenario-addressed test.

I think that, for the moment, I will remove this test, as it relies on other components that are already addressed.

In a future, I think it would be nice to enable integration tests that do not rely on any private API:

1. Enabling tests that are run via docker-compose, as it is actually being done.
2. Changing the components' constructors to not be constructed directly from a configuration file but allowing dependency injection.